samples/suit/ab/README.rst

```diff
@@ -0,0 +1,678 @@
+
+The suit-invoke sequence of the root manifest reads the ``BOOT_STATUS`` manifest variable value and acts upon it.
+If the application has been just updated, which is indicated by the ``CONFIRM_SET_A`` or ``CONFIRM_SET_B`` manifest variable set to ``JUST_INSTALLED``,
+and correctly validated by the suit-validate sequence, the confirmation variable is reset to ``NOT_CONFIRMED``.
+It is now the applications responsibility to set this variable to ``CONFIRMED``.
+Otherwise this variant will be considered as degraded on the next boot attempt.
+The following activity diagram presents the logic, implemented inside the suit-invoke sequence.
+
+.. figure:: img/ab_root_invoke.png
+
+   :alt: The suit-invoke sequence of the root manifest
+
+It is worth noticing that apart from regular application/radio/root manifest, the application recovery manifest is also defined.
+In the A/B sample there is no separate recovery binary, but just the recovery manifest is used.
+The recovery manifest is very similar to the root manifest, so in case of the root manifest damage, the recovery manifest serves as a backup logic.
+The recovery manifest logic deviates from the root manifest in two areas:
+
+* The boot preference variable is ignored and the manifest simply tries to boot first the variant ``A`` and then the variant ``B``.
+* The recovery logic always boots the application and radio in the "degraded" mode.
+
+User interface
+**************
+
+LED 0:
+    This LED is controlled by the application variant ``A``.
+    This LED is controlled as "active low", meaning it will turn on once the application is booted and blinks will be visible as short off periods.
+    The ``CONFIG_N_BLINKS`` Kconfig option sets the number of blinks.
+    The default of the Kconfig is set to ``1``, causing a single blink to indicate *Version 1*.
+    This value can be incremented to represent an update, such as setting it to ``2`` to indicate *Version 2*.
+    Using the ``-Dab_CONFIG_N_BLINKS`` command line option, the number of blinks can be set for both variants of the application.
+    If the application variant ``B`` is started, this LED will not blink.
+
+LED 1:
+    This LED is controlled by the application variant ``B``.
+    This LED is controlled as "active low", meaning it will turn on once the application is booted and blinks will be visible as short off periods.
+    Using the ``-Dab_slot_b_CONFIG_N_BLINKS`` command line option, it is possible to change the number of blinks just for the application variant ``B``.
+    If the application variant ``A`` is started, this LED will not blink.
+
+LED 2 and 3:
+    These LEDs indicate the status of manifests inside the device:
+
+    * If both LEDs are turned ``On``, the application is booted normally.
+    * If both LEDs blink simultaneously, the application is booted in a degraded mode (i.e. into an unconfirmed variant).
+    * If both LEDs blink out of phase, the application is booted using the recovery manifest.
```

Comment:
nit: I would use "blink in an alternating pattern" instead "out of phase", or "blink alternately", that is for better readability when comparing to degraded mode.